quantum/split_common/transactions.c

```diff
@@ -578,6 +581,43 @@ static void st7565_handlers_slave(matrix_row_t master_matrix[], matrix_row_t sla
 
 #endif  // defined(ST7565_ENABLE) && defined(SPLIT_ST7565_ENABLE)
 
+////////////////////////////////////////////////////
+// Haptic
+
+#if !defined(NO_ACTION_LAYER) && defined(SPLIT_HAPTIC_ENABLE)
+
+uint8_t split_haptic_play = 0xFF;
+
+static bool haptic_handlers_master(matrix_row_t master_matrix[], matrix_row_t slave_matrix[]) {
+    bool okay = transport_write(PUT_HAPTIC, &split_haptic_play, sizeof(split_haptic_play));
```

Comment:
If anything, the entire haptic config structure should be synced over, not *just* the mode.  Especially as the "enable" states isn't being synced.

Reply:
Any concerns about the size of the haptic structure when sync'd? I guess it's only 8 bytes, but split transport seems sensitive to size of data being transported, at least from reading the docs.

As the patch is structured, it's not a concern that the enable state isn't sync'd, since haptic events are triggered by haptic_play(), which would only be invoked if enable status is true.

Reply:
It should be fine.  Mostly, it gets synced if there are changes, but that shouldn't be too often.  And then forcibly synced on the forced timeout (defaults to 500ms). 

So for the sync, it shouldn't be too much.